client/src/components/HistoryViewer/HistoryViewerVersionList.js

```diff
@@ -1,14 +1,24 @@
 import React, { PureComponent } from 'react';
-import HistoryViewerHeading from 'components/HistoryViewer/HistoryViewerHeading';
-import HistoryViewerVersion from 'components/HistoryViewer/HistoryViewerVersion';
+import HistoryViewerHeading from './HistoryViewerHeading';
+import HistoryViewerVersion from './HistoryViewerVersion';
 import { versionType } from 'types/versionType';
 
 class HistoryViewerVersionList extends PureComponent {
+  /**
+   * Return a string of HTML class names for the table element
+   *
+   * @returns {string}
+   */
+  getClassNames() {
```

Comment:
We should be consistent here and use the `classnames` library.

`return classnames({ table: true, extraClass});`